components/webui/server/src/plugins/MongoSocketIoServer/index.ts

```
@@ -77,6 +81,37 @@ class MongoSocketIoServer {
+            const [firstAddress] = addresses;
+            if ("undefined" === typeof firstAddress) {
+                fastify.log.error("No addresses found for the provided host.");
+
+                return;
+            }
+
+            if (1 < addresses.length) {
+                fastify.log.warn(
+                    `Multiple addresses found for ${fastify.config.HOST}: ${
+                        JSON.stringify(addresses)}. ` +
+                    "Proxying all socket.io requests to the main server."
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)